mainProject/src/components/Users/common/product/sliderContent.js

```diff
@@ -5,7 +5,10 @@ class SliderContent extends React.Component {
 
 
     render() {
-        let cardList = this.props.products.map((product, index) => <Card key={product.productName + index} productInfo={product}></Card>);
+        let cardList;
+        if(this.props.products){
```

Comment:
ngược lại thì có view thông báo gì không em

Reply:
ngược lại thì có view thông báo ạ. Nó chỉ để tránh lỗi khi lần đầu render mà this.props.products không có dữ liệu nên không map được ạ.